docs/guide/content/user_guide/finding_data.md

```diff
@@ -67,6 +67,11 @@ If you prefer an integrated experience or are still exploring:
 
 Once a dataset is selected, OnDemand Loop presents its metadata and file listing so you can choose what to download into your project.
 
+!!! note
+
+    For **Dataverse** datasets, OnDemand Loop allows the user to select the desired dataset version, including the current `draft` if an
+    API key has been previously configured for that **Dataverse** installation.
+
```

Comment:
It reads like the API key is required for accessing all versions including Draft.
Lets update to clarify that the API key is only required for Draft:

` A repository API token is required to access and display draft versions.`

Reply:
fixed